src/renderer/components/message/MessageList.js

```diff
@@ -76,6 +77,10 @@ export default function MessageList (props) {
 
   const onClickSetupMessage = setupMessage => openDialog('EnterAutocryptSetupMessage', { setupMessage })
   const onShowDetail = message => openDialog('MessageDetail', { message, chat })
+  const onDelete = message => openDialog('ConfirmationDialog', {
+    message: 'Are you sure you want to delete message',
```

Comment:
Please add this to https://github.com/deltachat/deltachat-desktop/blob/master/_locales/_untranslated_en.json and use the translated message

Reply:
https://github.com/deltachat/deltachat-desktop#add-experimental-language-strings

Reply:
I will add the changes. Thanks for your feedback😊